frontend/pages/Parts/[...deviceHandleItemType].tsx

```diff
@@ -1,145 +1,11 @@
-import {
-   AppProviders,
-   AppProvidersProps,
-   Layout,
-   WithLayoutProps,
-   WithProvidersProps,
-} from '@components/common';
-import {
-   ProductListView,
-   ProductListViewProps,
-} from '@components/product-list';
-import { ALGOLIA_PRODUCT_INDEX_NAME } from '@config/env';
 import { serverSidePropsWrapper } from '@helpers/next-helpers';
-import {
-   decodeDeviceTitle,
-   decodeDeviceItemType,
-   encodeDeviceTitle,
-} from '@helpers/product-list-helpers';
-import { invariant } from '@ifixit/helpers';
-import { urlFromContext } from '@ifixit/helpers/nextjs';
-import { getGlobalSettings } from '@models/global-settings';
-import { findProductList } from '@models/product-list';
-import { getStoreByCode, getStoreList } from '@models/store';
-import { GetServerSideProps } from 'next';
-import { getServerState } from 'react-instantsearch-hooks-server';
+import { ProductListType } from '@models/product-list';
+import { getProductListServerSideProps } from '@templates/product-list';
 
-type PageProps = WithLayoutProps<ProductListViewProps>;
-type AppPageProps = WithProvidersProps<PageProps>;
-
-const getServerSidePropsInternal: GetServerSideProps<AppPageProps> = async (
-   context
-) => {
-   context.res.setHeader(
-      'Cache-Control',
-      'public, s-maxage=10, stale-while-revalidate=600'
-   );
-
-   const { deviceHandleItemType } = context.params || {};
-   const [deviceHandle, itemTypeHandle, ...rest] = Array.isArray(
-      deviceHandleItemType
-   )
-      ? deviceHandleItemType
-      : [];
-
-   invariant(typeof deviceHandle === 'string', 'device handle is required');
-   if (rest?.length > 0) {
-      return {
-         notFound: true,
-      };
-   }
-
-   const itemType = itemTypeHandle
-      ? decodeDeviceItemType(itemTypeHandle)
-      : null;
-
-   const deviceTitle = decodeDeviceTitle(deviceHandle);
-
-   const [globalSettings, stores, currentStore, productList] =
-      await Promise.all([
-         getGlobalSettings(),
-         getStoreList(),
-         getStoreByCode('us'),
-         findProductList(
-            {
-               deviceTitle: {
-                  eqi: deviceTitle,
-               },
-            },
-            itemType
-         ),
-      ]);
-
-   if (productList == null) {
-      return {
-         notFound: true,
-      };
-   }
-
-   if (productList.deviceTitle && productList.deviceTitle !== deviceTitle) {
-      const slug = itemTypeHandle ? `/${itemTypeHandle}` : '';
-      const canonicalDeviceTitle = encodeDeviceTitle(productList.deviceTitle);
-      return {
-         redirect: {
-            permanent: true,
-            destination: `/Parts/${canonicalDeviceTitle}${slug}`,
-         },
-      };
-   }
-
-   const title = `iFixit | ${productList.title}`;
```

Comment:
We now prefer `iFixit` for the title as opposed to `iFixit | Productlist Title`? I didn't see it mentioned anywhere, so just wanted to check.

Reply:
This is just a default value, which is overridden [here](https://github.com/iFixit/react-commerce/blob/main/frontend/components/product-list/MetaTags.tsx#L68).
I expect we'll do similar overrides for every template, so that value is only there to ensure that we have a default value.
You can check the preview to see that the behaviour is unchanged. 
I did this change to make it explicit that this is a default value, and to simplify the generation of layout props.